doc/en/user/source/community/gsr/installing.rst

```diff
@@ -0,0 +1,8 @@
+Installing the GeoServer JSON-LD extension
```

Comment:
This should be installing the GSR extension

Reply:
fixed